system/t12_api/snapshots.py

```diff
@@ -208,7 +213,7 @@ def check(self):
                  "Signing": DefaultSigningOptions,
             }
         )
-        self.check_equal(resp.json()['State'], TASK_SUCCEEDED)
+        self.check_task(task)
 
         self.check_equal(self.delete_task("/api/snapshots/" + snap2).json()['State'], TASK_FAILED)
         self.check_equal(self.delete_task("/api/snapshots/" + snap2, params={"force": "1"}).json()['State'], TASK_FAILED)
```

Comment:
Why not replacing occurences of `check_equal(..., TASK_FAILED)` with `check_task_fail` in this file ?

Reply:
fixed, thanks !  :)